pkg/operator2/idp.go

```diff
@@ -343,3 +346,12 @@ func encodeOrDie(obj runtime.Object) []byte {
 	}
 	return bytes
 }
+
+func trustedCABytes() []byte {
```

Comment:
are these additional to the system trust store? The file only includes some CAs, not all. Especially not possibly required intermediate ones.